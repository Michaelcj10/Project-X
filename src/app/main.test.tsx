import Main from "./main";
import * as React from "react";
import { render } from "@testing-library/react";

describe("<TeamInfo /> spec", () => {
  it("renders the component", () => {
    const container = render(<Main />);
    expect(container).toMatchSnapshot();
  });
});
