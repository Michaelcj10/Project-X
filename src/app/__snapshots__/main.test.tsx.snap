// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TeamInfo /> spec renders the component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
        <div>
            <header
                class="sc-AykKC fmcvyS"
            />
            <div
                class="sc-AykKE lmTlgN"
            >
                <div
                    class="hero-block"
                >
                    <svg
                        xmlns="http://www.w3.org/2000/svg"
                        viewBox="0 0 1440 320"
                    >
                        <path
                            fill="#fff"
                            fill-opacity="1"
                            d="M0,64L34.3,74.7C68.6,85,137,107,206,112C274.3,117,343,107,411,117.3C480,128,549,160,617,192C685.7,224,754,256,823,245.3C891.4,235,960,181,1029,170.7C1097.1,160,1166,192,1234,197.3C1302.9,203,1371,181,1406,170.7L1440,160L1440,320L1405.7,320C1371.4,320,1303,320,1234,320C1165.7,320,1097,320,1029,320C960,320,891,320,823,320C754.3,320,686,320,617,320C548.6,320,480,320,411,320C342.9,320,274,320,206,320C137.1,320,69,320,34,320L0,320Z"
                        />
                    </svg>
                    <div
                        class="row"
                    >
                        <div
                            class="twelve columns centered"
                        >
                            <div>
                                <h1>
                                    Elegant & 
                                    <br />
                                    creative solutions.
                                </h1>
                                <img
                                    id="dots"
                                />
                            </div>
                            <div>
                                <p>
                                    Website and application development with a difference.
                                </p>
                            </div>
                            <div>
                                <button
                                    class="sc-AykKD eaJiOE"
                                >
                                    Contact Us
                                </button>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <div
                class="sc-AykKG cYViYu"
            >
                <div
                    class="infographic-block container"
                >
                    <div
                        class="row"
                    >
                        <div
                            class="twelve columns centered"
                        >
                            <figure
                                class="parallax-outer custom-class"
                            >
                                <div
                                    class="parallax-inner"
                                >
                                    <h2>
                                        Puca Labs
                                    </h2>
                                    <img />
                                </div>
                            </figure>
                            <p>
                                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed augue quam, egestas vitae elementum eu, blandit vitae felis. Maecenas lobortis vehicula risus sit amet elementum.
                            </p>
                        </div>
                    </div>
                    <figure
                        class="parallax-outer custom-class"
                    >
                        <div
                            class="parallax-inner"
                        >
                            <div
                                class="sc-AykKF iSoVXx"
                            >
                                <div
                                    class="row"
                                >
                                    <div
                                        class="one-half column centered padded"
                                    >
                                        <div
                                            class="img-block greyed-on-touch"
                                        >
                                            <div>
                                                <img
                                                    alt="Mobile"
                                                />
                                            </div>
                                        </div>
                                    </div>
                                    <div
                                        class="one-half column centered padded"
                                    >
                                        <div>
                                            <h2>
                                                Creative designs
                                            </h2>
                                        </div>
                                        <div>
                                            <p>
                                                We make high quality , mobile responsive and SEO compliant websites. If you want to get full exposure for your business, or you just want a personal site to show off your holiday snaps, or wedding photos, we've got you covered!
                                            </p>
                                            <div
                                                class="dn-break"
                                            />
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </figure>
                    <figure
                        class="parallax-outer custom-class"
                    >
                        <div
                            class="parallax-inner"
                        >
                            <div
                                class="sc-AykKF iSoVXx"
                            >
                                <div
                                    class="row"
                                >
                                    <div
                                        class="one-half column centered padded"
                                    >
                                        <div>
                                            <h2>
                                                Mobile responsive
                                            </h2>
                                        </div>
                                        <div>
                                            <p>
                                                We make high quality , mobile responsive and SEO compliant websites. If you want to get full exposure for your business, or you just want a personal site to show off your holiday snaps, or wedding photos, we've got you covered!
                                            </p>
                                            <div
                                                class="dn-break"
                                            />
                                        </div>
                                    </div>
                                    <div
                                        class="one-half column centered padded"
                                    >
                                        <div
                                            class="img-block greyed-on-touch"
                                        >
                                            <div>
                                                <img
                                                    alt="Mobile"
                                                />
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </figure>
                    <figure
                        class="parallax-outer custom-class"
                    >
                        <div
                            class="parallax-inner"
                        >
                            <div
                                class="sc-AykKF iSoVXx"
                            >
                                <div
                                    class="row"
                                >
                                    <div
                                        class="one-half column centered padded"
                                    >
                                        <div
                                            class="img-block greyed-on-touch"
                                        >
                                            <div>
                                                <img
                                                    alt="Mobile"
                                                />
                                            </div>
                                        </div>
                                    </div>
                                    <div
                                        class="one-half column centered padded"
                                    >
                                        <div>
                                            <h2>
                                                Social integration
                                            </h2>
                                        </div>
                                        <div>
                                            <p>
                                                We make high quality , mobile responsive and SEO compliant websites. If you want to get full exposure for your business, or you just want a personal site to show off your holiday snaps, or wedding photos, we've got you covered!
                                            </p>
                                            <div
                                                class="dn-break"
                                            />
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </figure>
                </div>
            </div>
            <div
                class="sc-AykKH fMQdlv"
            />
            <div
                class="sc-AykKJ ectulF"
            >
                <div
                    class="row container"
                >
                    <div
                        class="four columns centered padded"
                    >
                        <div>
                            <svg
                                viewBox="0 0 512 512"
                                height="75"
                                width="75"
                                aria-hidden="true"
                                focusable="false"
                                fill="currentColor"
                                xmlns="http://www.w3.org/2000/svg"
                                color="#3f3d56"
                                class="sc-AykKI jYynPS"
                            >
                                <path
                                    fill="currentColor"
                                    d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm0 448c-110.5 0-200-89.5-200-200S145.5 56 256 56s200 89.5 200 200-89.5 200-200 200zm61.8-104.4l-84.9-61.7c-3.1-2.3-4.9-5.9-4.9-9.7V116c0-6.6 5.4-12 12-12h32c6.6 0 12 5.4 12 12v141.7l66.8 48.6c5.4 3.9 6.5 11.4 2.6 16.8L334.6 349c-3.9 5.3-11.4 6.5-16.8 2.6z"
                                />
                            </svg>
                            <br />
                        </div>
                        <p>
                            Speed
                        </p>
                        <p>
                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed  augue quam, egestas vitae elementum eu, blandit vitae felis. Maecenas lobortis vehicula risus sit amet elementum.
                        </p>
                        <img
                            id="dots"
                        />
                    </div>
                    <div
                        class="four columns centered padded"
                    >
                        <div>
                            <svg
                                viewBox="0 0 24 24"
                                height="75"
                                width="75"
                                aria-hidden="true"
                                focusable="false"
                                fill="currentColor"
                                xmlns="http://www.w3.org/2000/svg"
                                color="#3f3d56"
                                class="sc-AykKI fMYBqk"
                            >
                                <path
                                    d="M20 12c0-1.103-.897-2-2-2h-1V7c0-2.757-2.243-5-5-5S7 4.243 7 7v3H6c-1.103 0-2 .897-2 2v8c0 1.103.897 2 2 2h12c1.103 0 2-.897 2-2v-8zM9 7c0-1.654 1.346-3 3-3s3 1.346 3 3v3H9V7z"
                                />
                            </svg>
                            <br />
                        </div>
                        <p>
                            Security
                        </p>
                        <p>
                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed  augue quam, egestas vitae elementum eu, blandit vitae felis. Maecenas lobortis vehicula risus sit amet elementum.
                        </p>
                        <img
                            id="dots"
                        />
                    </div>
                    <div
                        class="four columns centered padded"
                    >
                        <div>
                            <svg
                                viewBox="0 0 512 512"
                                height="75"
                                width="75"
                                aria-hidden="true"
                                focusable="false"
                                fill="currentColor"
                                xmlns="http://www.w3.org/2000/svg"
                                color="#3f3d56"
                                class="sc-AykKI jYynPS"
                            >
                                <path
                                    fill="currentColor"
                                    d="M505.04 442.66l-99.71-99.69c-4.5-4.5-10.6-7-17-7h-16.3c27.6-35.3 44-79.69 44-127.99C416.03 93.09 322.92 0 208.02 0S0 93.09 0 207.98s93.11 207.98 208.02 207.98c48.3 0 92.71-16.4 128.01-44v16.3c0 6.4 2.5 12.5 7 17l99.71 99.69c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.59.1-33.99zm-297.02-90.7c-79.54 0-144-64.34-144-143.98 0-79.53 64.35-143.98 144-143.98 79.54 0 144 64.34 144 143.98 0 79.53-64.35 143.98-144 143.98zm27.11-152.54l-45.01-13.5c-5.16-1.55-8.77-6.78-8.77-12.73 0-7.27 5.3-13.19 11.8-13.19h28.11c4.56 0 8.96 1.29 12.82 3.72 3.24 2.03 7.36 1.91 10.13-.73l11.75-11.21c3.53-3.37 3.33-9.21-.57-12.14-9.1-6.83-20.08-10.77-31.37-11.35V112c0-4.42-3.58-8-8-8h-16c-4.42 0-8 3.58-8 8v16.12c-23.63.63-42.68 20.55-42.68 45.07 0 19.97 12.99 37.81 31.58 43.39l45.01 13.5c5.16 1.55 8.77 6.78 8.77 12.73 0 7.27-5.3 13.19-11.8 13.19h-28.1c-4.56 0-8.96-1.29-12.82-3.72-3.24-2.03-7.36-1.91-10.13.73l-11.75 11.21c-3.53 3.37-3.33 9.21.57 12.14 9.1 6.83 20.08 10.77 31.37 11.35V304c0 4.42 3.58 8 8 8h16c4.42 0 8-3.58 8-8v-16.12c23.63-.63 42.68-20.54 42.68-45.07 0-19.97-12.99-37.81-31.59-43.39z"
                                />
                            </svg>
                            <br />
                        </div>
                        <p>
                            SEO
                        </p>
                        <p>
                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed  augue quam, egestas vitae elementum eu, blandit vitae felis. Maecenas lobortis vehicula risus sit amet elementum.
                        </p>
                        <img
                            id="dots"
                        />
                    </div>
                </div>
            </div>
        </div>
    </div>
</body>,
  "container": <div>
    <div>
        <header
            class="sc-AykKC fmcvyS"
        />
        <div
            class="sc-AykKE lmTlgN"
        >
            <div
                class="hero-block"
            >
                <svg
                    xmlns="http://www.w3.org/2000/svg"
                    viewBox="0 0 1440 320"
                >
                    <path
                        fill="#fff"
                        fill-opacity="1"
                        d="M0,64L34.3,74.7C68.6,85,137,107,206,112C274.3,117,343,107,411,117.3C480,128,549,160,617,192C685.7,224,754,256,823,245.3C891.4,235,960,181,1029,170.7C1097.1,160,1166,192,1234,197.3C1302.9,203,1371,181,1406,170.7L1440,160L1440,320L1405.7,320C1371.4,320,1303,320,1234,320C1165.7,320,1097,320,1029,320C960,320,891,320,823,320C754.3,320,686,320,617,320C548.6,320,480,320,411,320C342.9,320,274,320,206,320C137.1,320,69,320,34,320L0,320Z"
                    />
                </svg>
                <div
                    class="row"
                >
                    <div
                        class="twelve columns centered"
                    >
                        <div>
                            <h1>
                                Elegant & 
                                <br />
                                creative solutions.
                            </h1>
                            <img
                                id="dots"
                            />
                        </div>
                        <div>
                            <p>
                                Website and application development with a difference.
                            </p>
                        </div>
                        <div>
                            <button
                                class="sc-AykKD eaJiOE"
                            >
                                Contact Us
                            </button>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <div
            class="sc-AykKG cYViYu"
        >
            <div
                class="infographic-block container"
            >
                <div
                    class="row"
                >
                    <div
                        class="twelve columns centered"
                    >
                        <figure
                            class="parallax-outer custom-class"
                        >
                            <div
                                class="parallax-inner"
                            >
                                <h2>
                                    Puca Labs
                                </h2>
                                <img />
                            </div>
                        </figure>
                        <p>
                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed augue quam, egestas vitae elementum eu, blandit vitae felis. Maecenas lobortis vehicula risus sit amet elementum.
                        </p>
                    </div>
                </div>
                <figure
                    class="parallax-outer custom-class"
                >
                    <div
                        class="parallax-inner"
                    >
                        <div
                            class="sc-AykKF iSoVXx"
                        >
                            <div
                                class="row"
                            >
                                <div
                                    class="one-half column centered padded"
                                >
                                    <div
                                        class="img-block greyed-on-touch"
                                    >
                                        <div>
                                            <img
                                                alt="Mobile"
                                            />
                                        </div>
                                    </div>
                                </div>
                                <div
                                    class="one-half column centered padded"
                                >
                                    <div>
                                        <h2>
                                            Creative designs
                                        </h2>
                                    </div>
                                    <div>
                                        <p>
                                            We make high quality , mobile responsive and SEO compliant websites. If you want to get full exposure for your business, or you just want a personal site to show off your holiday snaps, or wedding photos, we've got you covered!
                                        </p>
                                        <div
                                            class="dn-break"
                                        />
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </figure>
                <figure
                    class="parallax-outer custom-class"
                >
                    <div
                        class="parallax-inner"
                    >
                        <div
                            class="sc-AykKF iSoVXx"
                        >
                            <div
                                class="row"
                            >
                                <div
                                    class="one-half column centered padded"
                                >
                                    <div>
                                        <h2>
                                            Mobile responsive
                                        </h2>
                                    </div>
                                    <div>
                                        <p>
                                            We make high quality , mobile responsive and SEO compliant websites. If you want to get full exposure for your business, or you just want a personal site to show off your holiday snaps, or wedding photos, we've got you covered!
                                        </p>
                                        <div
                                            class="dn-break"
                                        />
                                    </div>
                                </div>
                                <div
                                    class="one-half column centered padded"
                                >
                                    <div
                                        class="img-block greyed-on-touch"
                                    >
                                        <div>
                                            <img
                                                alt="Mobile"
                                            />
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </figure>
                <figure
                    class="parallax-outer custom-class"
                >
                    <div
                        class="parallax-inner"
                    >
                        <div
                            class="sc-AykKF iSoVXx"
                        >
                            <div
                                class="row"
                            >
                                <div
                                    class="one-half column centered padded"
                                >
                                    <div
                                        class="img-block greyed-on-touch"
                                    >
                                        <div>
                                            <img
                                                alt="Mobile"
                                            />
                                        </div>
                                    </div>
                                </div>
                                <div
                                    class="one-half column centered padded"
                                >
                                    <div>
                                        <h2>
                                            Social integration
                                        </h2>
                                    </div>
                                    <div>
                                        <p>
                                            We make high quality , mobile responsive and SEO compliant websites. If you want to get full exposure for your business, or you just want a personal site to show off your holiday snaps, or wedding photos, we've got you covered!
                                        </p>
                                        <div
                                            class="dn-break"
                                        />
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </figure>
            </div>
        </div>
        <div
            class="sc-AykKH fMQdlv"
        />
        <div
            class="sc-AykKJ ectulF"
        >
            <div
                class="row container"
            >
                <div
                    class="four columns centered padded"
                >
                    <div>
                        <svg
                            viewBox="0 0 512 512"
                            height="75"
                            width="75"
                            aria-hidden="true"
                            focusable="false"
                            fill="currentColor"
                            xmlns="http://www.w3.org/2000/svg"
                            color="#3f3d56"
                            class="sc-AykKI jYynPS"
                        >
                            <path
                                fill="currentColor"
                                d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm0 448c-110.5 0-200-89.5-200-200S145.5 56 256 56s200 89.5 200 200-89.5 200-200 200zm61.8-104.4l-84.9-61.7c-3.1-2.3-4.9-5.9-4.9-9.7V116c0-6.6 5.4-12 12-12h32c6.6 0 12 5.4 12 12v141.7l66.8 48.6c5.4 3.9 6.5 11.4 2.6 16.8L334.6 349c-3.9 5.3-11.4 6.5-16.8 2.6z"
                            />
                        </svg>
                        <br />
                    </div>
                    <p>
                        Speed
                    </p>
                    <p>
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed  augue quam, egestas vitae elementum eu, blandit vitae felis. Maecenas lobortis vehicula risus sit amet elementum.
                    </p>
                    <img
                        id="dots"
                    />
                </div>
                <div
                    class="four columns centered padded"
                >
                    <div>
                        <svg
                            viewBox="0 0 24 24"
                            height="75"
                            width="75"
                            aria-hidden="true"
                            focusable="false"
                            fill="currentColor"
                            xmlns="http://www.w3.org/2000/svg"
                            color="#3f3d56"
                            class="sc-AykKI fMYBqk"
                        >
                            <path
                                d="M20 12c0-1.103-.897-2-2-2h-1V7c0-2.757-2.243-5-5-5S7 4.243 7 7v3H6c-1.103 0-2 .897-2 2v8c0 1.103.897 2 2 2h12c1.103 0 2-.897 2-2v-8zM9 7c0-1.654 1.346-3 3-3s3 1.346 3 3v3H9V7z"
                            />
                        </svg>
                        <br />
                    </div>
                    <p>
                        Security
                    </p>
                    <p>
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed  augue quam, egestas vitae elementum eu, blandit vitae felis. Maecenas lobortis vehicula risus sit amet elementum.
                    </p>
                    <img
                        id="dots"
                    />
                </div>
                <div
                    class="four columns centered padded"
                >
                    <div>
                        <svg
                            viewBox="0 0 512 512"
                            height="75"
                            width="75"
                            aria-hidden="true"
                            focusable="false"
                            fill="currentColor"
                            xmlns="http://www.w3.org/2000/svg"
                            color="#3f3d56"
                            class="sc-AykKI jYynPS"
                        >
                            <path
                                fill="currentColor"
                                d="M505.04 442.66l-99.71-99.69c-4.5-4.5-10.6-7-17-7h-16.3c27.6-35.3 44-79.69 44-127.99C416.03 93.09 322.92 0 208.02 0S0 93.09 0 207.98s93.11 207.98 208.02 207.98c48.3 0 92.71-16.4 128.01-44v16.3c0 6.4 2.5 12.5 7 17l99.71 99.69c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.59.1-33.99zm-297.02-90.7c-79.54 0-144-64.34-144-143.98 0-79.53 64.35-143.98 144-143.98 79.54 0 144 64.34 144 143.98 0 79.53-64.35 143.98-144 143.98zm27.11-152.54l-45.01-13.5c-5.16-1.55-8.77-6.78-8.77-12.73 0-7.27 5.3-13.19 11.8-13.19h28.11c4.56 0 8.96 1.29 12.82 3.72 3.24 2.03 7.36 1.91 10.13-.73l11.75-11.21c3.53-3.37 3.33-9.21-.57-12.14-9.1-6.83-20.08-10.77-31.37-11.35V112c0-4.42-3.58-8-8-8h-16c-4.42 0-8 3.58-8 8v16.12c-23.63.63-42.68 20.55-42.68 45.07 0 19.97 12.99 37.81 31.58 43.39l45.01 13.5c5.16 1.55 8.77 6.78 8.77 12.73 0 7.27-5.3 13.19-11.8 13.19h-28.1c-4.56 0-8.96-1.29-12.82-3.72-3.24-2.03-7.36-1.91-10.13.73l-11.75 11.21c-3.53 3.37-3.33 9.21.57 12.14 9.1 6.83 20.08 10.77 31.37 11.35V304c0 4.42 3.58 8 8 8h16c4.42 0 8-3.58 8-8v-16.12c23.63-.63 42.68-20.54 42.68-45.07 0-19.97-12.99-37.81-31.59-43.39z"
                            />
                        </svg>
                        <br />
                    </div>
                    <p>
                        SEO
                    </p>
                    <p>
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed  augue quam, egestas vitae elementum eu, blandit vitae felis. Maecenas lobortis vehicula risus sit amet elementum.
                    </p>
                    <img
                        id="dots"
                    />
                </div>
            </div>
        </div>
    </div>
</div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
